Apps/WebClient/src/ClientApp/src/components/timeline/laboratory.vue

```diff
@@ -65,7 +65,7 @@ $radius: 15px;
             <b-col class="entryTitle">
                 <b-row class="justify-content-between">
                     <b-col cols="auto">
-                        <strong>{{ entry.summaryTestType }}</strong>
```

Comment:
The changes look good. However, it seems that the body.vue is still using the summaryTestType which is causing the build failed. Here are some logs:
"ERROR in /opt/az/_work/13/s/Apps/WebClient/src/ClientApp/src/components/calendar/body.vue(330,55):
  330:55 Property 'summaryTestType' does not exist on type 'LaboratoryTimelineEntry'."

Reply:
Good catch! I've updated it